README.md

```diff
@@ -2,207 +2,436 @@
 
 A revamped USA.gov site using Drupal 9 and Cloud Foundry
 
-## Initial Project Setup
-At the start of the project and at any other time you wish to "reset" your local development environment you may run the init script to prep any necessary files and rebuild containers. Starting the containers will initially lead you to an empty Drupal site.
+<a name="readme-top"></a>
+
+## Table of contents
+
+- [USAgov 2021](#usagov-2021)
+  - [Table of contents](#table-of-contents)
+  - [Prerequisites](#prerequisites)
+  - [Setting Up the Project](#setting-up-the-project)
+    - [Initialization](#initialization)
+    - [Database Setup](#database-setup)
+    - [Media files Setup](#media-files-setup)
+    - [Access the Drupal Portal](#access-the-drupal-portal)
+  - [How to re-run the website](#how-to-re-run-the-website)
+    - [Using the terminal](#using-the-terminal)
+    - [Using Docker Desktop](#using-docker-desktop)
+  - [Theme Lint Guidelines](#theme-lint-guidelines)
+  - [Project Restart/Reset](#project-restartreset)
+    - [Docker Cleanup](#docker-cleanup)
+  - [Update Database](#update-database)
+  - [Starting on a new ticket](#starting-on-a-new-ticket)
+  - [Continuing Work](#continuing-work)
+  - [Tickets and Branching](#tickets-and-branching)
+  - [Single Item Config Export](#single-item-config-export)
+  - [USAgovTheme](#usagovtheme)
+  - [Export Database](#export-database)
+  - [Export Config](#export-config)
+  - [Import Config](#import-config)
+  - [Build and Deploy procedure](#build-and-deploy-procedure)
+- [Troubleshooting](#troubleshooting)
+  - [If cms password is not accepted:](#if-cms-password-is-not-accepted)
+  - [More info on Cloud Foundry \& Cloud.gov](#more-info-on-cloud-foundry--cloudgov)
+
+
+## Prerequisites
+
+Before starting the project setup install/download the following:
+
+**Note: To download these tools, you must have administrator rights. If you do not have them and you're using a GSA computer, you can request admin rights [here](https://gsa.servicenowservices.com/sp/?id=sc_cat_item&sys_id=d75222fa1b9c641cb1f620eae54bcb20).**
+
+
+* [Homebrew](https://docs.brew.sh/Installation)
+* [Git](https://formulae.brew.sh/formula/git#default)
+* [Php](https://formulae.brew.sh/formula/php#default)
+* [Composer](https://formulae.brew.sh/formula/composer#default)
+* [Docket Desktop](https://docs.docker.com/desktop/install/mac-install/)
+* [Drupal](https://www.digitalocean.com/community/tutorials/how-to-develop-a-drupal-9-website-on-your-local-machine-using-docker-and-ddev#step-1-mdash-installing-ddev)
+* [jq](https://formulae.brew.sh/formula/jq#default)
+* [npm](https://radixweb.com/blog/installing-npm-and-nodejs-on-windows-and-mac)
+* [Visual Studio Code](https://code.visualstudio.com/download)
```

Comment:
VS Code is not _required_, just a code editor/IDE of some kind, VSCode is just the most popular right now. :)